### DISCUSSION TITLE
when pv backup, totalBytes is displayed too large.

### DISCUSSION THREAD

**Discussion by woogeee** [vmware-tanzu/velero#6207] [Community support Q&A]
status:
  phase: InProgress
  progress:
    bytesDone: 359685853906
    totalBytes: 565662186742
  startTimestamp: "2023-04-27T22:41:59Z"

The size of this PV is 100Gi.

**Comment by woogeee** [comment]
I found that snapshot files are included in backup file.
I want exclude specific files backing up.
How can i use restic option(--exclude-file) on velero backup?

**Comment by draghuram** [comment]
Hi @woogeee, I don't think it is possible to exclude some PV files from a Velero backup. It is always going to be entire PV backup though after the first one, only changes should be backed up.

**Comment by woogeee** [comment]
Hi @draghuram, I also know that the capacity decreases after the first backup. But in many cases it is throwing errors when backing up snapshot files. All backup failures are caused by files related to snapshots. Backup capacity also increases by more than five times. So I want to back up excluding snapshot files.
